vividus-soft-assert/src/test/java/org/vividus/softassert/model/KnownIssueTests.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.softassert.model;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.vividus.softassert.issue.KnownIssueType;
+
+public class KnownIssueTests
+{
+    private KnownIssue knownIssue = new KnownIssue("id", KnownIssueType.AUTOMATION, false);
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = "done")
+    void testStatusClosedFalse(String status)
+    {
+        knownIssue.setStatus(status);
+        Assertions.assertFalse(knownIssue.isClosed());
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"closed", "CLOSED"})
+    void testStatusClosedTrue(String status)
```

Comment:
join this test and previous one and use `CsvSource` or `MethodSource`

Reply:
is it okay to use assertEquals in that case?

Reply:
any correct way to assert the expected behavior is allowed